alt-tab-macos/logic/Preferences.swift

```diff
@@ -79,7 +80,7 @@ class Preferences {
             tabKeyCode = try UInt16(value).orThrow()
         case "metaKey":
             let p = try metaKeyMacro.labelToMacro[value].orThrow()
-            metaKeyCode = p.preferences.0
+            metaKeyCode = UInt16(p.preferences.0)
```

Comment:
Ive kept metaKeyCode and tabKeyCode to be UInt16 so we don't need to cast during the often occurring listen events in Keyboard.keyboardHandler() which would be then from NSEvent.keyCode (UInt16) to HIToolbox.Events Symbols (Int).

Reply:
This is casting the values declared as `Int` from line 44 above to `UInt16`. I think just use `UInt16` everywhere and you will not need to cast here, no?

Reply:
Yeah, somewhere the cast has to happen, either at the top in this file where the Mapping occurs or at this code line. The apple provided symbols are old and are just declared as Int.